src/fluent-plugin-mdsd/gemspec-template

```diff
@@ -0,0 +1,30 @@
+Gem::Specification.new do |s|
+    s.name = "fluent-plugin-mdsd"
+    s.version = "GEMVERSION"
```

Comment:
this file is a template. it will be updated to a proper version through 'Makefile'. so that the version can be centrally controlled across the product (e.g. debian pkg also needs the same version)

---
In reply to: [114165536](https://github.com/Azure/fluentd-plugin-mdsd/pull/26#discussion_r114165536) [](ancestors = 114165536)